.github/workflows/test.yml

```diff
@@ -14,11 +14,11 @@ jobs:
       matrix:
         platform:
           - os: ubuntu-latest
-            rs: 1.79
+            rs: 1.80.0
```

Comment:
would you care to also change these 2 beautiful lines, badge https://github.com/near/near-sdk-rs/blob/eda17d5e2705ec71a1c704b483bae86dd79e53cd/README.md?plain=1#L15 and 
MSRV mention https://github.com/near/near-sdk-rs/blob/eda17d5e2705ec71a1c704b483bae86dd79e53cd/README.md?plain=1#L324

Reply:
Done